extensions/sql-migration/package.json

```diff
@@ -2,14 +2,14 @@
   "name": "sql-migration",
   "displayName": "%displayName%",
   "description": "%description%",
-  "version": "0.0.2",
+  "version": "0.0.3",
   "publisher": "Microsoft",
   "preview": true,
   "license": "https://raw.githubusercontent.com/Microsoft/azuredatastudio/main/LICENSE.txt",
   "icon": "images/extension.png",
   "engines": {
     "vscode": "^1.25.0",
```

Comment:
Honestly you could just put this as * even since it doesn't really matter what that is